docs/cron/backup.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+BACKUP_LIST=(
+    "/var/www-prod/app/webroot/img/profiles_128"
+    "/var/www-prod/app/webroot/img/profiles_36"
+    "/home/debian/tatowiki/tatowiki/app"
+    "/var/www-audio/sentences"
+    "/home/debian/dump/db.sql.gz"
+)
+
+GDRIVE="/home/debian/gdrive/gdrive"
+
+BACKUP_DIR="backupdir"
+mkdir -p $BACKUP_DIR
+
+# in Mbs
+CHUNK_SIZE=32
+CHUNK_SIZE=$(echo "$CHUNK_SIZE*1024*1024" | bc)
+
+KEEP_NUM=2
+
+DATE=$(date -u +%F)
+
+prune(){
+    search_name=$1
+    keep_num=$2
+
+    flst=$($GDRIVE list --query "name contains '$search_name'" --order "createdTime desc" | tail -n +2);
+    curr_num=$(echo "$flst" | wc -l)
+    if [[ $curr_num -gt $keep_num ]]; then
+        dlst=$(diff <(echo "$flst") <(echo "$flst" | head -n $keep_num) |\
+         tail -n +2 | sed 's/< //g' | awk '{print $1}'
+        )
+
+        echo "Pruning $(echo "$dlst" | wc -l) files related to $search_name ..."
+        IFS=$'\n' dlst=($dlst)
+        for d in ${dlst[@]}; do
+            echo "Deleting $d"
+            $GDRIVE delete "$d"
+        done
+    fi
+}
+
+echo "Starting gdrive backup script at $DATE ..."
+
+for i in "${BACKUP_LIST[@]}"; do
+    # check for file or dir
+    if [[ "$(echo $i | grep -o '\.' | uniq)" == "." ]]; then
```

Comment:
Is this supposed to check if $i is a file? If yes, I suggest using `[[ -f "$i" ]]` instead (and `-d` for directory check).